CHANGES.md

```diff
@@ -4,13 +4,21 @@
 
 - **irmin**:
   - Add sanity checks when creating `Irmin.Type` records, variants and enums
```

Comment:
This entry is getting quite verbose, but we can review it before releasing (and after any future bugs you find :slightly_smiling_face:).